rust-peer/src/main.rs

```diff
@@ -364,3 +428,91 @@ async fn read_or_create_identity(path: &Path) -> Result<identity::Keypair> {
 
     Ok(identity)
 }
+
+// Simple file exchange protocol
```

Comment:
Mind moving this to another module just to not bloat `main.rs` as much?

Reply:
Makes sense: b45a786